src/App.tsx

```diff
@@ -1,61 +1,40 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { getUserById } from './services/user';
+import { Todo } from './types/Todo';
+import { PostForm } from './PostForm';
```

Comment:
The `PostForm` component is imported directly, but according to the task requirements, data should be prepared in the `App` component and passed down as props. Ensure that `PostForm` receives necessary data as props from `App`.